streamlit_modal/__init__.py

```diff
@@ -1,115 +1,160 @@
+from uuid import uuid4
 from contextlib import contextmanager
 
+from deprecation import deprecated
 import streamlit as st
 import streamlit.components.v1 as components
 
 
-def is_open():
-    return st.session_state.get('modal_is_open', False)
+class Modal:
 
+    def __init__(self, title, padding=100, max_width=None, background_color='#fff', key=None):
+        self.title = title
+        self.padding = padding
+        self.backgroud_color = background_color
```

Comment:
Seems `background_color` is not used?

Reply:
removed in f75545600d023a331606a13833f6c197e9ded73d